app/frontend/wallet/cardano-wallet.js

```diff
@@ -285,7 +287,7 @@ const CardanoWallet = async (options) => {
 
   async function getChangeAddress() {
     // if we used all available addresses return random address from the available ones
```

Comment:
remove please this old comment, it's misleading since now we always return a random address